gprofiler/profilers/java.py

```diff
@@ -783,6 +910,16 @@ def _init_java_safemode(self, java_safemode: str) -> None:
                 f" --java-safemode={JavaSafemodeOptions.JAVA_EXTENDED_VERSION_CHECKS}"
             )
 
+    def _init_java_flag_collection(self, java_collect_jvm_flags: str) -> Union[JavaFlagCollectionOptions, List[str]]:
+        if java_collect_jvm_flags == "":
```

Comment:
Why is this a legit value? UX wise this seems like "none" to me - I passed a list and it was empty. It should have the same behavior as `none`.